Description: Module::Build 0.40 is not available in Debian/wheezy

--- a/Build.PL
+++ b/Build.PL
@@ -12,7 +12,7 @@
         'Test::More' => 0,
     },
     configure_requires => {
-        'Module::Build' => 0.40,
+        'Module::Build' => 0,
     },
     dist_version_from => 'lib/Data/HAL.pm',
     dynamic_config => 0,
--- a/META.json
+++ b/META.json
@@ -33,7 +33,7 @@
       },
       "configure" : {
          "requires" : {
-            "Module::Build" : "0.4"
+            "Module::Build" : "0"
          }
       },
       "runtime" : {
--- a/META.yml
+++ b/META.yml
@@ -12,7 +12,7 @@
   Test::More: 0
   strictures: 0
 configure_requires:
-  Module::Build: 0.4
+  Module::Build: 0
 dynamic_config: 0
 generated_by: 'Module::Build version 0.4007, CPAN::Meta::Converter version 2.132830'
 license: perl
